# AprilTag 3 code parameters
# Find descriptions in apriltag/include/apriltag.h:struct apriltag_detector
#                      apriltag/include/apriltag.h:struct apriltag_family
tag_family:        'tag36h11' # options: tagStandard52h13, tagStandard41h12, tag36h11, tag25h9, tag16h5, tagCustom48h12, tagCircle21h7, tagCircle49h12  
#用于选择标签种类,默认使用’tag36h11’，其泛用性也最好。
tag_threads:       2                # default: 2       
#设置多线程，允许核心APRILTAG 2并行计算的最大线程数。
tag_decimate:      1.0          # default: 1.0       
#最小图像分辨率，以位置精度换取识别速度
tag_blur:          0.0                 # default: 0.0      
#tag_blur> 0模糊图像，tag_blur  < 0锐化图像
tag_refine_edges:  1          # default: 1        
#以算力换取计算精度，值为1时计算成本比较低。
tag_debug:         0                # default: 0        
#值为1时，将中间图保存像到~/.ros
max_hamming_dist:  2    # default: 2 
#一般都将值设置为2，当值大于等于3时，会消耗大量内存。尽可能选择最大的值。
publish_tf:        true            # default: false    
#发布tf坐标
